guacamole/src/main/webapp/translations/ja.json

```diff
@@ -0,0 +1,867 @@
+{
+    
+    "NAME" : "Japanese",
+    
+    "APP" : {
+
+        "NAME"    : "Apache Guacamole",
+        "VERSION" : "1.0.0",
```

Comment:
These values should not be duplicated, here - if you look at the other translations, they are left out and should only be present in the `en.json` file.